src/olympia/conf/dev/settings.py

```diff
@@ -118,7 +118,7 @@
         'content_host': 'https://stable.dev.lcip.org',
         'oauth_host': 'https://oauth-stable.dev.lcip.org/v1',
         'profile_host': 'https://stable.dev.lcip.org/profile/v1',
-        'redirect_url': 'https://amo.addons-dev.allizom.org/fxa-authenticate',
+        'redirect_url': 'https://addons-dev.allizom.org/api/v3/accounts/authenticate/?config=amo', # noqa
```

Comment:
if you want to change this check with ops the value of `AMO_FXA_CLIENT_ID` and verify the `redirect_url` specified on FxA.

Reply:
This is this one: https://oauth-stable.dev.lcip.org/console/client/cb6a5ba26feea430